src/services/formService.js

```diff
@@ -0,0 +1,5 @@
+import api from 'api';
+
+export default {
+  fetchFormCategories: () => api.get('form_categories')
+};
```

Comment:
Maybe follow convention here, See [User Service](https://github.com/rootstrap/open-form-admin-webapp/blob/master/src/services/userService.js) for an example

Reply:
It just felt a bit weird to follow the other style, because we are using hooks to avoid class boilerplate, but also we are using classes in this place. Can I leave like this and the refactor the other using class to look more like this?

Reply:
As long as it's consistent, it's ok. But there are a few services that are statefull (like our fetch wrapper for example). In those cases classes are easier to understand. So i'm not sure you'll be able to migrate all of them 🤷‍♂

Reply:
where's that? I meant to make the changes in this folder, but I can take a look to this one you mention after I complete some features

Reply:
check `/api/ApiService.js`

Reply:
where's is the state there? I only see you pass some variables when you create an instance, but you can keep those on the same file without needing a class

Reply:
interceptors are stored as an instance variable.